integration-tests/support/pages/hacbs/OverviewTabPage.ts

```diff
@@ -0,0 +1,8 @@
+import { over } from 'cypress/types/lodash';
+import { overviewTabPagePO } from '../../pageObjects/hacbs-po';
+
+export class OverviewTabPage {
+    goTocomponentsTab() {
```

Comment:
`goToComponentsTab`, not enough camelCase :)

Reply:
Noted and updated, thanks!